website/docs/modeling-data.md

```diff
@@ -192,7 +192,9 @@ Two kind of relationships between models are allowed:
 
 Relationships must define the related `type` and may optionally define their
 `inverse`, which should correspond to the name of a relationship on the related
-model. Inverse relationships should be defined when relationships must be kept
+model. Multiple values may be expressed as elements of an array.
```

Comment:
Since the previous sentence mentions both `type` and `inverse`, I think it's helpful to clarify that only `type` can be expressed as an array: 

```suggestion
model. Multiple `type` values may be expressed as elements of an array.
```